package.json

```diff
@@ -50,7 +50,7 @@
   },
   "packageManager": "npm@10.2.5+sha256.8002e3e7305d2abd4016e1368af48d49b066c269079eeb10a56e7d6598acfdaa",
   "engines": {
-    "node": ">=12.13.0"
+    "node": ">=14.0.0"
```

Comment:
Technically, this is a breaking change (should generate v2 of the plugin). But in practice, v12 is too old so maybe it’s OK. If there are any plans to release anything breaking soon, might be worth bumping Node to v18 given the opportunity. v16 and below are EOL.